torch/csrc/jit/operator_upgraders/utils.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+#include <c10/util/Optional.h>
+#include <torch/csrc/jit/operator_upgraders/version_map.h>
+#include <iostream>
+#include <regex>
+#include <string>
+#include <vector>
+
+namespace torch {
+namespace jit {
+
+static c10::optional<UpgraderEntry> findUpgrader(
+    std::vector<UpgraderEntry> upgraders_for_schema,
+    int current_version) {
+  // we want to find the entry which satisfies following two conditions:
+  //    1. the version entry must be greater than current_version
+  //    2. Among the version entries, we need to see if the current version
+  //       is in the upgrader name range
+  auto pos = std::find_if(
+      upgraders_for_schema.begin(),
+      upgraders_for_schema.end(),
+      [current_version](const UpgraderEntry& entry) {
+        return entry.version_bump > current_version;
```

Comment:
My understanding is that for `div_Tensor_0_3`, the loaded model operator version needs to be within [0,3]

Reply:
I thought it was more like versions between 0 and 3 need to be upgraded with the current upgrader. So div_Tensor_0_3 implies there was a version bump which is greater than 3. @gmagogsfm What do you think about this check?